charts/consul/values.yaml

```diff
@@ -2245,6 +2245,15 @@ terminatingGateways:
   gateways:
     - name: terminating-gateway
 
+# Configuration settings for the webhook-cert-manager
```

Comment:
can we expand on this a bit and talk about what the cert manager does? Just so folks aren't confused about what exact certs it's managing.

Reply:
@kschoche What is your take on what I added?